pkg/network/protocols/testutil/serverutils.go

```diff
@@ -35,7 +35,7 @@ func RunDockerServer(t testing.TB, serverName, dockerPath string, env []string,
 	ctx, cancel := context.WithCancel(context.Background())
 	t.Cleanup(cancel)
 
-	cmd := exec.CommandContext(ctx, "docker-compose", "-f", dockerPath, "up")
+	cmd := exec.CommandContext(ctx, "docker-compose", "-f", dockerPath, "up", "--force-recreate", "--remove-orphans")
```

Comment:
why did you add `force-recreate`?

Reply:
I was getting errors about the network `testdata-network-1` not being present. It seemed like some sort of race with the cleanup.